worker/pkg/workflows/datasync/activities/activities.go

```diff
@@ -634,7 +633,15 @@ func parseJavascriptForColumnName(jsCode, targetWord, replacementWord string) st
 
 func constructJavascriptCode(jsCode, col string) string {
 	if jsCode != "" {
-		return fmt.Sprintf(`function fn%[2]s(value){%s};const input = benthos.v0_msg_as_structured();const output = { ...input };output["%[2]s"] = fn%[2]s(input["%[2]s"]);benthos.v0_msg_set_structured(output)`, jsCode, col)
+		return fmt.Sprintf(`
+function fn%s(value){
+  %s
+};
+const input = benthos.v0_msg_as_structured();
+const output = { ...input };
+output["%s"] = fn%s(input["%s"]);
+benthos.v0_msg_set_structured(output);
+`, col, jsCode, col, col, col)
```

Comment:
this might be fine for now but the reason why in my examples I used numbers for the fn name is that idk if all json keys are compatible as function names. maybe we leave it for now until we find an issue.